src/test/java/POM/PlaylistPage.java

```diff
@@ -0,0 +1,70 @@
+package POM;
+
+import com.github.javafaker.Faker;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.util.Locale;
+
+public class PlaylistPage extends BasePage{
+    public PlaylistPage(WebDriver givenDriver) {
+
+        super(givenDriver);
+    }
+    //LOCATORS
+    By plusBtn = By.cssSelector("[data-testid='sidebar-create-playlist-btn']");
+    By createNewPlaylist = By.cssSelector("[data-testid='playlist-context-menu-create-simple']");
+    By playlistNameInput = By.cssSelector(".create input");
+    By plNameInput = By.cssSelector("[id='songResultsWrapper'] [placeholder='Playlist name']");
+    By checkHeader = By.cssSelector("#playlistWrapper h1");
+    By delPlaylistBtn = By.cssSelector(".btn-delete-playlist");
+    By toVerifyBanner = By.cssSelector(".success");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```